plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -182,6 +187,62 @@ class Bazel(
         }
     }
 
+    /**
+     * Collect the archive overrides from the `MODULE.bazel` file in [workingDir]. The overrides are collected with
+     * Buildozer and are returned as a map from package names to archive overrides. An empty map is returned if no
+     * override is defined.
+     */
+    private fun getArchiveOverrides(workingDir: File): Map<String, ArchiveOverride> {
+        val process = ProcessCapture(
+            BUILDOZER_COMMAND,
+            "print module_name urls integrity patches",
+            "//MODULE.bazel:%archive_override",
+            workingDir = workingDir
+        )
+
+        require(process.isSuccess) {
+            "Failed to get archive overrides from 'buildozer': ${process.stderr}"
+        }
+
+        // If an optional attribute is missing, buildozer outputs first a warning line and then the result with the
+        // value "(missing"):
+        // $ buildozer 'print integrity patch_cmds ' //MODULE.bazel:%archive_override
+        //      rule "//.:bazel-archive-override" has no attribute "patch_cmds"
+        //      sha256-3B9PcEylbj1e3Zc/mKRfBIfQ8oxonQpXuiNhEhSLGDM= (missing)
+        return process.stdout.lines().filterNot {
+            it.isEmpty() || it.contains("has no attribute")
+        }.associate { line ->
+            val results = line.split(' ', limit = 4)
+            val moduleName = results[0]
+            val urlsAsString = results[1]
+            val integrity = results[2]
+            val patchesAsString = results[3]
```

Comment:
Prefer a destructing declaration:

    val (moduleName, urlsAsString, integrity, patchesAsString) = line.split(' ', limit = 4)

Reply:
![image](https://github.com/user-attachments/assets/bd5e7faf-2648-4933-a38e-715dda6bf556)

Well `detekt` is passing in CI tests but there is a warning in Intellij. What do you think?

Reply:
I'd simply suppress that inspection hint here. Or maybe even globally raise he limit to 4, as 4 is probably a common number to destruct for if you think e.g. of vectors etc.

Reply:
It's already 4 in the Detekt configuration...so it's just some weird behavior from Intellij